Automattic/sensei
# Don't show incorrect answer feedback if grade is zero

Fixes #5472 
Fixes #6239 

### Changes proposed in this Pull Request

* Hide answer feedback when a question has a zero grade since it won't contribute to the final score.
* Do not show Right/Wrong buttons for multiple choice and true/false questions when the question has zero grade.

### Testing instructions

**Test 1**

1. Create a lesson and add a quiz with auto grading turned on.
2. Add a multiple choice or true/false question
3. Make those questions worth 0 Points
4. Notice the Right/Wrong button disappears. Change the grade between 0 and any number to see the behaviour.

**Test 2**
1. Create a quiz with a Single Line, Multiline, and/or File Upload Question, and Automatic Grading turned on.
2. Make those questions worth 0 Points
3. Take the quiz.
4. The questions worth 0 points should not show the correct/incorrect answer feedback.

### Screenshot / Video

<img width="945" alt="Screenshot 2022-12-20 at 12 50 27" src="https://user-images.githubusercontent.com/26160845/208660475-b55d91ff-6b91-4349-ae93-9a3e07c439e2.png">
